frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-card border="md" class="d-flex flex-row" @click="console.log()">
+    <v-card-text class="border-e-md d-flex flex-column justify-space-between align-center">
+      <h2 class="text-h6 text-center">
+        {{ equipment.name }}
+      </h2>
+      <v-progress-circular
+        :model-value="equipment.oee"
+        size="150"
+        width="20"
+        :color="getColorState(equipment.oee)"
+        class="text-h6 mt-2">
+        <span>
+          <span :style="{fontSize: '1rem'}" class="font-weight-medium">OEE</span>
+          <br>
+          {{Math.round(equipment.oee)}}%
+        </span>
+      </v-progress-circular>
+    </v-card-text>
+
+    <v-card-text class="pa-0">
+      <v-card
+        v-for="(metric, metricIndex) in metrics"
+        :key="metric.label"
+        class="pa-0 d-flex flex-column justify-center"
+        :class="metricIndex +1 === metrics.length ? '' : 'border-b-md'"
+        height="33%"
+        flat
+        tile
+      >
+        <v-sheet :color="getColorState(metric.value)" :style="{position: 'absolute', width: `${metric.value}%`, height: '100%', zIndex: -1, opacity: 0.5}"/>
```

Comment:
To simplify code I'd recommend some CSS classes instead of the non-variable styles, but not too important.